src/libsync/configfile.cpp

```diff
@@ -262,6 +344,24 @@ void ConfigFile::restoreGeometry(QWidget *w)
 #endif
 }
 
+void ConfigFile::saveGeometry(QWindow *w)
+{
+#ifndef TOKEN_AUTH_ONLY
+    ASSERT(!w->objectName().isNull());
```

Comment:
@er-vin Isn't it necessary to check **w** is not null prior to calling **objectName()** on it? Also, maybe it makes sense to use **Q_ASSERT** instead of **ASSERT**?

Reply:
Admittedly I went along with what was in the saveGeometry(QWidget *w) overload. I'd rather keep improvements like the one you propose here for a different PR or such (where both would be changed together).